Others/Dijkstra.java

```diff
@@ -56,7 +58,7 @@ public static void main(String[] args) {
         public final String v1, v2;
         public final int dist;
 
-        public Edge(String v1, String v2, int dist) {
+        public Edge(@NonNull String v1, @NonNull String v2, @NonNegative int dist) {

```

Comment:
`@NonNull is not needed, see the explanation below.

Reply:
Done!